.github/workflows/release-post-merge.yml

```diff
@@ -5,20 +5,20 @@
 name: Create Release
 
 on:
-  pull_request:
-    branches:
-      - pre-release/**
-      - release/**
-    types:
-      - closed
-  workflow_dispatch:
+  push:
+    tags:
+      - 'v*'
```

Comment:
i guess a regex would be too much here? 😬 just worried this will catch a tag like `very-nice-tag`.

Reply:
I am not sure that danger avails for the cost. Regex is all but standard, and a proper semver regexp is costly to test. I do not even trust the regex by **act** would be the same as the actual regex used by GH CI actions. If we have a regex that is good enough and is known to work with GitHub CI I would not mind.

Reply:
good points